src/main/java/sg/ncl/MainController.java

```diff
@@ -254,6 +268,42 @@ public String testbedInformation(Model model) throws IOException {
         return "testbedInformation";
     }
 
+    // get all the nodes' status
+    // there are three types of status
+    // "free" : node is free
+    // "in_use" : node is in use
+    // "reload" : node is in process of freeing or unknown status
+    // "reserved" : node is pre-reserved for a project
+    @RequestMapping("/testbedNodesStatus")
+    public String testbedNodesStatus(Model model) throws IOException {
+
+        Map<MachineType, List<Map<String, String>>> nodesStatus = getNodesStatus();
+        EnumMap<MachineType, Map<String, Long>> nodesStatusCount = new EnumMap<>(MachineType.class);
+
+        // loop through each of the machine type
+        // tabulate the different nodes type
+        // count the number of different nodes status, e.g. SYSTEMX = { FREE = 10, IN_USE = 11, ... }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)